api/issue.go

```diff
@@ -173,7 +173,7 @@ type UpdateSchemaGhostDetail struct {
 type UpdateSchemaGhostContext struct {
 	// UpdateSchemaDetail is the details of schema update.
 	// When a project is in tenant mode, there should be one item in the list.
-	UpdateSchemaDetailList []*UpdateSchemaDetail `json:"updateSchemaDetailList"`
+	UpdateSchemaDetailList []*UpdateSchemaGhostDetail `json:"updateSchemaDetailList"`
```

Comment:
nit: since the parent struct is called `UpdateSchemaGhostContext`, how about we just name the field
```go
DetailList []*UpdateSchemaGhostDetail `json:"updateSchemaGhostDetailList"`
```

Reply:
Done.